packages/model-viewer/src/features/controls.ts

```diff
@@ -240,6 +240,8 @@ export declare interface ControlsInterface {
   getCameraOrbit(): SphericalPosition;
   getCameraTarget(): Vector3D;
   getFieldOfView(): number;
+  getMinimumFieldOfView(): number|undefined;
+  getMaximumFieldOfView(): number|undefined;
```

Comment:
Our types are a little weird, but this is actually always defined (it has default values).

Reply:
See my comment in the impl